docs/source/models.rst

```diff
@@ -2425,12 +2425,12 @@ Pipeline Type                     Input Type                     Output Type
 Instructional Text Generation     str or List[str]               str or List[str]
 Conversational                    str or List[str]               str or List[str]
 Summarization                     str or List[str]               str or List[str]
-Text Classification               str or List[str]               str or List[str]
+Text Classification               str or List[str]               pd.DataFrame
```

Comment:
Is it possible to include the dataframe schema like this?

```
pd.DataFrame (dtypes: {foo: int, bar: float})
```

Reply:
great idea :)